repl.js

```diff
@@ -1002,60 +1002,72 @@ import * as os from "os";
         }
         print_rec(a);
     }
-    
-    function extract_directive(a) {
-        var pos;
-        if (a[0] !== '\\')
-            return "";
-        for (pos = 1; pos < a.length; pos++) {
-            if (!is_alpha(a[pos]))
-                break;
-        }
-        return a.substring(1, pos);
-    }
 
-    /* return true if the string after cmd can be evaluted as JS */
-    function handle_directive(cmd, expr) {
-        if (cmd === "h" || cmd === "?" || cmd == "help") {
+    /* return true if the string was a directive */
+    function handle_directive(a) {
+        var pos;
+        if (a === "?") {
             help();
-        } else if (cmd === "load") {
-            var filename = expr.substring(cmd.length + 1).trim();
-            if (filename.lastIndexOf(".") <= filename.lastIndexOf("/"))
-                filename += ".js";
-            std.loadScript(filename);
+            return true;
+        }
+        if (a[0] !== '\\' && a[0] !== '.')
```

Comment:
If we are switching to dot directives, it feeels weird to get `.quit` in the docs, for example, but for `\quit` to also work.

Also, `\quit()` and `\quit123` also work, which is arguably wrong.

Reply:
I agree `.quit()` and `\quit123` should be rejected.  Will do.

I switched to `.` directives because they are more readable and that's what node uses, I kept backslash for muscle memory, but I should probably remove it at some point.

Reply:
👍